integration_tests/semantic_segmentation_tests/experiment.py

```diff
@@ -55,7 +55,9 @@ def get_path(part):
                                            .with_raster_source(label_path) \
                                            .build()
 
+        vector_output = {'uri': '*', 'mode': 'buildings'}
```

Comment:
Rather than use '*' to indicate that it should be autogenerated, I think `uri` should have a default value of None which is treated as "autogenerate" to be more consistent with the rest of RV.

Reply:
Resolved.